src/AnnotationThread.js

```diff
@@ -57,21 +56,26 @@ class AnnotationThread extends EventEmitter {
         this.annotatedElement = data.annotatedElement;
```

Comment:
Maybe a bit of cleanliness around extracting each data prop to pass to properties on this?

ie)
```
const {api, container, id, and, so, on} = data;
```